### ISSUE TITLE
Homepage: YouTube video image is missing alt text

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/justgetsimple#50]
Just noting that I fixed this by changing it from a UABB Video Module to a regular Video Module.

**Comment by JacquelineOmollo** [comment]
done
